public/app/features/alerting/unified/utils/datasource.ts

```diff
@@ -43,7 +43,7 @@ export function getRulesDataSources() {
   }
 
   return getAllDataSources()
-    .filter((ds) => RulesDataSourceTypes.includes(ds.type) && ds.jsonData.manageAlerts !== false)
```

Comment:
Revert. By default it should be enabled

Reply:
I think this is maybe the only blocker in this PR

Reply:
I'm very confused with this change, is not supposed to be the same logic?

Reply:
> Revert. By default it should be enabled

what do you mean?